docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```diff
@@ -219,9 +232,13 @@ components:
     StellarId:
       type: object
       description:
-        StellarId's are objects that identify an on-chain entity. Both end users and SEP-31 senders can be identified
-        with StellarId's. Id's for SEP-31 senders will never include a `memo` or `memo_type`.
+        StellarId's are objects that identify a SEP entity. Both end-users and SEP-31 senders/receivers can be identified
```

Comment:
I'm not sure "SEP Entity" is any more clear than "on-chain entity". Maybe we should just be explicit and say "Stellar ID's are objects that identify end-users and SEP-31 senders (they won't identify receivers).

Reply:
> I'm not sure "SEP Entity" is any more clear than "on-chain entity"

To me, `on-chain entity` is the same as `stellar account id`, which is not exactly what this is for, right?

> "Stellar ID's are objects that identify end-users and SEP-31 senders(...)

Ok

> (they won't identify receivers)

You mean they don't identify the `Receiving Anchors`, right? The receiving end-user can be identified with it.

--
I've updated the description in https://github.com/stellar/java-stellar-anchor-sdk/pull/412/commits/df6d62803935be72225784b835eabd1797eacfab